Library/Formula/jemalloc.rb

```diff
@@ -16,8 +16,5 @@ class Jemalloc < Formula
   def install
     system "./configure", "--disable-debug", "--prefix=#{prefix}", "--with-jemalloc-prefix="
     system "make", "install"
-
-    # This otherwise conflicts with google-perftools
-    mv "#{bin}/pprof", "#{bin}/jemalloc-pprof"
```

Comment:
For the reviewer: this is no longer needed because the tool was [renamed upstream from `pprof` to `jeprof`](https://github.com/jemalloc/jemalloc/releases/tag/4.0.0).